condor/condor_meter

```diff
@@ -811,66 +733,25 @@ classad_double_re = re.compile("^(\w{1,255}) = ([+-]? *(?:\d{1,30}\.?\d{0,30}|\.
 classad_string_re = re.compile("^(\S+) = \"(.*)\"$")
 classad_catchall_re = re.compile("^(\S+) = (.*)$")
 def fd_to_classad(fd):
-    if g_has_classad:
-        buffer = ''
-    else:
-        classad = caselessDict()
+    buffer = ''
     for lineOrig in fd.readlines():
         line = lineOrig.strip()
-        if g_has_classad:
-           if not line:
-               if hasattr(classadLib, 'parseOne'):
-                   yield add_unique_id(classadLib.parseOne(buffer))
-               else:
-                   yield add_unique_id(classadLib.parseOld(buffer))
-               buffer = ''
-           else:
-               buffer += lineOrig
-           continue
-        m = classad_bool_re.match(line)
-        if m:
-            attr, val = m.groups()
-            if val.lower().find("true") >= 0:
-                classad[attr] = True
-            else:
-                classad[attr] = False
-            continue
-        m = classad_int_re.match(line)
-        if m:
-            attr, val = m.groups()
-            classad[attr] = int(val)
-            continue
-        m = classad_double_re.match(line)
-        if m:
-            attr, val = m.groups()
-            classad[attr] = float(val)
-            continue
-        m = classad_string_re.match(line)
-        if m:
-            attr, val = m.groups()
-            classad[attr] = str(val)
-            continue
-        m = classad_catchall_re.match(line)
-        if m:
-            attr, val = m.groups()
-            classad[attr] = str(val)
-            continue
         if not line:
-            yield add_unique_id(classad)
-            classad = caselessDict()
-            continue
-        DebugPrint(2, "Invalid line in ClassAd: %s" % line)
-
-    if g_has_classad:
-        if hasattr(classadLib, 'parseOne'):
-            yield add_unique_id(classadLib.parseOne(buffer))
+            if hasattr(classadLib, 'parseOne'):
+                yield add_unique_id(classadLib.parseOne(buffer))
+            else:
+                yield add_unique_id(classadLib.parseOld(buffer))
+            buffer = ''
         else:
-            yield add_unique_id(classadLib.parseOld(buffer))
+            buffer += lineOrig
+
+    if hasattr(classadLib, 'parseOne'):
+        yield add_unique_id(classadLib.parseOne(buffer))
     else:
-        yield add_unique_id(classad)
+        yield add_unique_id(classadLib.parseOld(buffer))
```

Comment:
We should be able to get rid of the check for `parseOne` availability and fallback to `parseOld`. `parseOne` has been available in condor since 8.3 (http://research.cs.wisc.edu/htcondor/manual/v8.3/6_7Python_Bindings.html).